mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.tsx

```diff
@@ -0,0 +1,71 @@
+import { ThemedScrollView, ThemedText } from '@components/themed'
+import { AccordionContent, WalletAccordion } from '@components/WalletAccordion'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+
+export function LiquidityMiningFaq (): JSX.Element {
+  const faqContent: AccordionContent[] = [
+    {
+      title: translate('components/LiquidityMiningFaq', 'How do I supply liquidity?'),
+      content: [{
+        text: translate('components/LiquidityMiningFaq', 'After deciding which pool pair you would like to supply liquidity to, click ADD LIQUIDITY and enter the input amount for either of the tokens in the pair. \n\nOnce an amount is entered on one side of the pair, the app automatically balances the amount of the other side, based on the ratio of current liquidity in the pool.'),
+        type: 'paragraph'
+      }]
+    },
+    {
+      title: translate('components/LiquidityMiningFaq', 'How do I send tokens from my DFI wallet?'),
+      content: [{
+        text: translate('components/LiquidityMiningFaq', 'Tokens on DeFiChain can only be sent to DFI addresses only. This means that you should not send directly to an exchange or a third party platform that does not support DeFiChain. \n\nSimilarly, liquidity tokens can only be sent to another DFI wallet address.'),
+        type: 'paragraph'
+      }]
+    },
+    {
+      title: translate('components/LiquidityMiningFaq', 'What are some exchanges and platforms I can send to?'),
+      content: [{
+        text: translate('components/LiquidityMiningFaq', '1. Kucoin\n2. Bittrex\n3. Transak\n4. Cake DeFi \n\nView more on [DeFiChain.com](https://defichain.com/dfi/)'),
```

Comment:
Is it suppose to be a link just like this?

View more on [DeFiChain.com](https://defichain.com/dfi/)

Reply:
Resolved with the wallet team